NEWS

```diff
@@ -1,3 +1,100 @@
+WiredTiger release 2.6.1, 2015-05-13
+------------------------------------
+
+The WiredTiger 2.6.1 release contains new features, minor API changes and bug
+fixes.
+
+New features:
+
+* Add the ability to disable write ahead logging at a per-table granularity.
+  Accessed via log=(enabled) configuration for WT_SESSION::create API.
+  Partial logging has serious implications for recovery, it should be used
+  with caution.
+  refs #1989
+
+* Add the ability to asynchronously flush transactions via a new
+  WT_SESSION::transaction_sync API.
+  refs WT-1908, #1943
+
+* Add the ability to create a named in-memory snapshot via a new
+  WT_SESSION::snapshot API.
+  refs WT-1839
+
+* Move the sync configuration setting from WT_SESSION::begin_transaction to
+  WT_SESSION::commit_transaction. Some applications don't know whether they
+  need to sync a transaction to disk until they come to commit.
+  refs WT-1908
+
+
+Other noteworthy changes:
+
+* Fix several bugs related to syncing files for checkpoint durability.
+  refs WT-1944
+
+* Fix a bug in the internal random number generator, where concurrent calls
+  could lead to invalid sequences. Never seen in the wild.
+
+* Fix compiler warnings for LZ4 implementation on Windows.
+  refs #2006
+
+* Fix a performance issue with WT_SESSION::compact, where it would spend a
+  long time compacting tables that had no space to reclaim.
+  refs WT-1953
+
+* Fix a bug in the WiredTiger command line utility where it could create a
+  base configuration file for an existing database.
+  refs WT-1943
+
+* Fix a build problem where recent versions of RedHat would fail to detect
+  posix_memalign presence correctly.
+  refs WT-1951
+
+* Fix several problems with how we create, recover and backup databases.
+  Related to order of creation and differences between Windows and Posix
+  file semantic differences.
```

Comment:
`Windows and Posix file semantic differences` -> `Windows and POSIX file system semantics`?